.github/workflows/workflow.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
     strategy:
       matrix:
         os: [ "ubuntu-latest", "macos-latest" ]
-        opa: [ "v0.42.2" ]
+        opa: [ "v0.44.0" ]
```

Comment:
do we want to bump this to 0.57.1 (same as GK 3.14)?

Reply:
@sozercan - At this moment we are still testing against GK 3.12 (n-1), so I would recommend we not go higher than `v0.49.2` at this exact moment.  That said, we can bump the CI to GK 3.13 once 3.14 is released.  `v0.44.0` has the advantage of supporting back to GK v3.10.

More context: https://github.com/open-policy-agent/gatekeeper-library/pull/429#discussion_r1376749615

Reply:
I see the GK 3.14 release just went up, so I'll open a PR to update the CI in a moment.

Reply:
The CI update to GK 3.14 (and retaining 3.13) is now merged!

@sozercan - As this is a matrix value, what if we **add** `"0.57.1"`?  That way we  can test against both the GK 3.14 opa version, but still maintain the signal for older versions.

Reply:
sgtm

Reply:
Do you want me to do something here?

Reply:
Hi @anderseknert - Sorry for the ambiguity, I just suggested the change based on the outcome.

Reply:
No worries! I've just pushed that change.

Reply:
Thanks! Running checks with both versions looks good.